### ISSUE TITLE
[Bug?]: Incompatibility with node version 18

### ISSUE THREAD

**Issue by OtterTax** [redwoodjs/redwood#8884]
### What's not working?

I cannot run `yarn create redwood-app ./redwoodblog` using node verion 18.  The program aborts with the following error:
```
error expo-cli@5.4.11: The engine "node" is incompatible with this module. Expected version ">=12 <=16". Got "18.12.1"
```

### How do we reproduce the bug?

Run `create redwood-app` with node version 18.16.1 and yarn version 1.22.19.

### Are you interested in working on this?

- [ ] I'm interested in working on this

**Comment by Josh-Walker-GM** [comment]
Hey @OtterTax thanks for raising this issue and trying out redwood! I'm not really sure but https://github.com/yarnpkg/yarn/issues/8732 might be helpful here? If not let me know and I can try and take a closer look at reproducing your exact error.

**Comment by OtterTax** [comment]
Thanks for getting back to me, Josh-Walker-GM.

The issue you link to indicates a problem with `create-strapi-app`, and I could find no trace of that on my system.  I deleted the yarn cache directory at `$HOME/.cache/yarn`, but still had no luck.

I was finally able to get around the problem by following one of the other suggestions in the thread you pointed me to.  Instead of using yarn, I installed with `npx create-redwood-app ./redwoodblog` and everything seemed to work fine.

I appreciate your help.
